website/src/content/deployment-examples/cloudflare.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Deploying to Cloudflare Pages
+---
+
+This deployment example refers to [Cloudflare Pages](https://developers.cloudflare.com/pages/).
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [Cloudflare](https://dash.cloudflare.com) account.
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, make sure to provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationId) that you got when you configured the Custom Application in the Merchant Center.
+
+Moreover, you need to provide the [production URL](/api-reference/application-config#envproductionurl) from your Cloudflare Pages project. You can keep the standard Cloudflare Pages URL `https://<project>.pages.dev` or provide your own custom domain.
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<project>.pages.dev"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the application ID, or the project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
```

Comment:
```suggestion
In case you want to avoid hardcoding certain values, for example the application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
```